.gitignore

```diff
@@ -50,3 +50,6 @@ Desktop.ini
 
 # Mac crap
 .DS_Store
+
+# NetBeans files
+/nbproject/
```

Comment:
This IDE / OS specific stuff should be in your own global gitignore, not at the project level.